packages/core-app-api/src/app/types.ts

```diff
@@ -213,6 +214,20 @@ export type AppOptions = {
     }
   >;
 
+  /**
+   * A map of plugin package names to internal groups (as entity refs defaulting
+   * to kind Group).
+   *
+   * The package name (key) can contain wildcards and will be matched using
+   * minimatch.
+   */
+  pluginOwners?: Record<string, string>[];
```

Comment:
I'm feeling that a use-case that is a bit tricky to implement right now is to link to internal slack channels or equivalent for the different teams. Perhaps the value here should be a bit more elaborate than just the owner entity refs? That does tie into the discussion just below about different ways to structure this API surface though. Another option in mind is stuff like this:

```ts
createApp({
  pluginDecorators: [
    PluginDecorators.byPluginIds({
      ids: ['catalog', 'scaffolder'],
      info: {
        owner: 'infrared', // potential shorthand to avoid full entity ref here
        links: [{ title: 'Slack', url: '...' }]
      },
    })
  ]
})
```

You're basically left with only the info decorator, but provide some utilities to make it easier to implement common use-cases. That kind of surface I think tends to be a bit easier to evolve.

Reply:
I find the lack of link between a catalog group entity and its internal slack channel (or equivalent) is a bigger problem, and one that shouldn't be fixed here specifically.

If there is a way to decorate (in a processor in the catalog, or _generically_ in the frontend app) communication links to groups, then we could piggy-back on that here. There are certainly more places where that would be very useful. The group entity pages themselves, e.g.

Reply:
The last commit adds group links on the BackstageApp, generic to the whole app. Configurable easily from e.g. a yaml file, like plugin ownership metadata.